app/src/main/res/layout/fragment_listener.xml

```diff
@@ -4,11 +4,49 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
+    <Button
+        android:id="@+id/btnSetting_Listener"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Setting"
```

Comment:
It's better to specify null instead of transparent background color as follows:
```suggestion
        android:text="Setting"
        android:background="@null"
```

Reply:
I didn't know about `@null` ! Thanks for your feedback :)
Then,
```suggestion
        android:background="@null"
        android:layout_marginRight="30dp"
```
how about this?